cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/PartitionLoadParameters.java

```diff
@@ -63,6 +65,11 @@ protected void initParameters() throws UnsupportedEncodingException {
     _entries = ParameterUtils.entries(_request);
     _minValidPartitionRatio = ParameterUtils.minValidPartitionRatio(_request);
     _allowCapacityEstimation = ParameterUtils.allowCapacityEstimation(_request);
+    try {
```

Comment:
If the provided broker id is invalid, I feel a better approach would be return an alert to user. 
In current code base, if we do not swallow the Exception here, it would be capture by the [error handling logic in `AbstractParameter` specifically for user parameter issue](https://github.com/linkedin/cruise-control/blob/cde921adb571f5c50f3afdb30770eddc29143c3b/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/AbstractParameters.java#L54).

Reply:
Currently, the endpoint can be used without specifying the broker id. If we do surface the error to the user, I think this would break backward compatibility of the endpoint. Is that okay?

Reply:
I see,  so this catch makes `brokerid` optional for this endpoint. looks good.